app/models/related-item-contributor.js

```diff
@@ -9,20 +9,25 @@ const Validations = buildValidations({
     validator('presence', {
       presence: true,
       message: 'Related Item contributor must have a name',
-    })
+    }),
+    validator(function(value, options) { 
+      console.log("RIC INLINE CALLED")
+      return true
+    }),
   ],
   contributorType: [
     validator('presence', {
       presence: true,
       message: 'Contributors must include a contributor type',
-      disabled: computed('model.name', function () {
-        return isBlank(this.model.get('name'));
-      })
+      // disabled: computed('name', function () {
+      //   console.log("VALIDATOR COMPUTED DETECTED NAME CHANGE");
+      //   return isBlank(this.name);
+      // })
     }),
     validator('contributor-type', {
```

Comment:
If I'm not mistaken, this validator is still active. one needs to remove the statement completely to de-activate it. I think you just removed the condition which to disable it. Which actually I think you still need in case that the user doesn't input any value in contributoName. @jrhoads @digitaldogsbody

Reply:
The validators inside this comment don't run automatically (something weird with the combination of ember-data-model-fragments and emper-cp-validations and the multi-level nesting of fragments. Instead, I am calling the validations manually from the parent object - which is where the disabling happens, so it is no longer needed here.

Reply:
ok, then maybe the issue is there because some of the validation for type seem to still be active, even on new dois that have nothing.

- https://app.frontapp.com/open/msg_huabfth
- https://app.frontapp.com/open/msg_huacjdh
- https://app.frontapp.com/open/msg_huabllh
- https://app.frontapp.com/open/msg_huad4np
- https://app.frontapp.com/open/msg_huad8k5
- [https://app.frontapp.com/open/msg_huabd3p ](https://app.frontapp.com/open/msg_huabd3p)